app/views/window/clients/vispdats/_list.haml

```diff
@@ -1,9 +1,23 @@
 %h1
   .pull-right
     - if @client.vispdats.in_progress.none?
-      = link_to polymorphic_path(vispdats_path_generator, client_id: @client.id), class: 'btn btn-primary', title: 'New', data: { method: :post } do
-        %span.icon-plus
-        New VI-SPDAT
+      .btn-group
+        %button.btn.btn-default.dropdown-toggle(type='button' data-toggle='dropdown')
+          New VI-SPDAT
+          %span.caret
+        %ul.dropdown-menu
+          %li
+            = link_to polymorphic_path(vispdats_path_generator, client_id: @client.id, type: 'GrdaWarehouse::Vispdat::Youth'), data: { method: :post } do
```

Comment:
We don't generally use font-awesome in this project, I'll see about adding them via the icomoon setup.

Reply:
Check `/style_guide/icon_font` for icomoon versions of these.  You should be able to use `.icon-group` `.icon-individual` and `.icon-child` I believe.